Core/Dialog/PSPOskDialog.cpp

```diff
@@ -892,14 +892,14 @@ int PSPOskDialog::NativeKeyboard() {
 			defaultText.assign(u"VALUE");
 
 		// There's already ConvertUCS2ToUTF8 in this file. Should we use that instead of the global ones?
-		System_InputBoxGetString(::ConvertUCS2ToUTF8(titleText), ::ConvertUCS2ToUTF8(defaultText), [&](bool result, const std::string &value) {
+		System_InputBoxGetString(::ConvertUCS2ToUTF8(titleText), ::ConvertUCS2ToUTF8(defaultText), [&](const std::string &value, int iValue) {
 			std::lock_guard<std::mutex> guard(nativeMutex_);
 			if (nativeStatus_ != PSPOskNativeStatus::WAITING) {
 				return;
 			}
 
 			nativeValue_ = value;
-			nativeStatus_ = result ? PSPOskNativeStatus::SUCCESS : PSPOskNativeStatus::FAILURE;
+			nativeStatus_ = iValue ? PSPOskNativeStatus::SUCCESS : PSPOskNativeStatus::FAILURE;

```

Comment:
This is just confusing since `iValue` is never used.

-[Unknown]

Reply:
Hm yeah, that was from an earlier version of the change that I didn't follow through completely on. Will fix.